DiffusionPreprocessing/scripts/eddy_postproc.sh

```diff
@@ -19,17 +19,17 @@ datadir=${workingdir}/data
 
 echo "Generating eddy QC report in ${workingdir}/qc"
 rm -r ${workingdir}/qc
-qc_command="${FSLDIR}/bin/eddy_quad "
-qc_command+="${eddydir}/eddy_unwarped_images "
-qc_command+="-idx ${eddydir}/index.txt "
-qc_command+="-par ${eddydir}/acqparams.txt "
-qc_command+="-m ${eddydir}/nodif_brain_mask.nii.gz "
-qc_command+="-b ${eddydir}/Pos_Neg.bvals "
-qc_command+="-g ${eddydir}/Pos_Neg.bvecs "
-qc_command+="-o ${workingdir}/qc "
-qc_command+="-f ${workingdir}/topup/topup_Pos_Neg_b0_field.nii.gz "
-qc_command+="-v "
-${qc_command}
+qc_command=("${FSLDIR}/bin/eddy_quad")
+qc_command+=("${eddydir}/eddy_unwarped_images")
+qc_command+=(-idx "${eddydir}/index.txt")
+qc_command+=(-par "${eddydir}/acqparams.txt")
+qc_command+=(-m "${eddydir}/nodif_brain_mask.nii.gz")
+qc_command+=(-b "${eddydir}/Pos_Neg.bvals")
+qc_command+=(-g "${eddydir}/Pos_Neg.bvecs")
+qc_command+=(-o "${workingdir}/qc")
+qc_command+=(-f "${workingdir}/topup/topup_Pos_Neg_b0_field.nii.gz")
+qc_command+=(-v)
+${qc_command[@]}
```

Comment:
This line should be `"${qc_command[@]}"`, using @ inside double quotes is a special syntx that causes the argument separation to be done by the array elements, rather than by whitespace.

Reply:
This type of syntax is why I don't like writing in bash...